src/RRule.php

```diff
@@ -2066,19 +2067,24 @@ static protected function i18nLoad($locale, $fallbackLocale = 'en_US')
 	 * intl extension is required.
 	 *
 	 * @param array  $opt
-	 * @param string $fallbackLocale
 	 *
 	 * @return string
 	 */
-	public function humanReadable(array $opt = array(), $fallbackLocale = 'en_US')
+	public function humanReadable(array $opt = array())
 	{
 		if ( self::$intl_loaded === null ) {
 			self::$intl_loaded = extension_loaded('intl');
 		}
 
+		$locale = setlocale(LC_MESSAGES, 0);
+		if ( self::$intl_loaded ) {
+			$locale = \Locale::getDefault();
+		}
+
```

Comment:
Here I would maybe handle the case of the special locale "C". So if locale == C, then locale becomes "en" for example.